test/sql/relocate_extension.sql

```diff
@@ -52,19 +52,19 @@ SELECT AVG(temp) AS avg_tmp, "testSchema0".time_bucket('1 day', time, INTERVAL '
 
 -- testing drop_chunks START
 -- show_chunks and drop_chunks output should be the same
-\set QUERY1 'SELECT "testSchema0".show_chunks(older_than => \'2017-03-01\'::timestamp, hypertable => \'test_ts\')::REGCLASS::TEXT'
+\set QUERY1 'SELECT "testSchema0".show_chunks(older_than => \'2017-03-01\'::timestamp, hypertable_or_cagg => \'test_ts\')::REGCLASS::TEXT'
```

Comment:
```suggestion
\set QUERY1 'SELECT "testSchema0".show_chunks(\'test_ts\', \'2017-03-01\'::timestamp)::REGCLASS::TEXT'
```

Reply:
Testing different parameter order is intentional.

Reply:
Not sure why this is necessary. This is something that PG handles, so there is no reason to test that different parameter orders work.

Changing the order to avoid the keywords will also avoid having to change the names of the parameters if we decide to use different names or correct typos.

Reply:
If the test needs a cleanup than I suggest we do that separately. I am just doing the changes necessary for this PR.